website/sidebars.json

```diff
@@ -127,7 +127,7 @@
       {
         "type": "subcategory",
         "label": "Hooks",
-        "ids": ["usecolorscheme", "usewindowdimensions"]
+        "ids": ["platformcolor", "usecolorscheme", "usewindowdimensions"]
```

Comment:
What's the reason behind listing `PlatformColor` in `Hooks` section not `APIs`?

Reply:
I was under the impression that's where it belongs because it's a function and operates similarly to usecolorscheme - if that's incorrect however, where do you think it should go?

Reply:
I was not digging into this too deep, but if what you said is correct and code structure of `PlatformColor` is basically a hook then it violates the [Rules of Hooks](https://reactjs.org/docs/hooks-custom.html) because all hooks must start with ["use"](https://reactjs.org/docs/hooks-faq.html#what-exactly-do-the-lint-rules-enforce). If it's true this should be addressed in RN repo.

If it's not a hook, page should be listed in `APIs` section as I wrote earlier.

Reply:
Hmm, @tom-un - should this have been "usePlatformColor"? If not I'll move it out of hooks.

Reply:
Alsto it seems that despite `PlatformColor` is in code base no entry in [CHANGELOG](https://github.com/react-native-community/releases/blob/master/CHANGELOG.md) mention about releasing it. Are you sure it was included in `0.62` (I'm asking because you also have updated `0.62` docs)?

If it is a part of `0.62` release changelog should be updated.

Reply:
@kikisaints, its an API not a hook.

Reply:
Thanks for checking. I moved it out of the versioned docs and out of the hooks section.